Firebase/Auth/Source/AuthProviders/OAuth/FIROAuthCredential_Internal.h

```diff
@@ -46,11 +48,13 @@ NS_ASSUME_NONNULL_BEGIN
     @param IDToken The ID Token associated with the credential being created.
     @param accessToken The access token associated with the credential being created.
     @param pendingToken The pending token associated with the credential being created.
+    @param secret The secret associated with the credential being created.
  */
 - (instancetype)initWithProviderID:(NSString *)providerID
                            IDToken:(nullable NSString *)IDToken
                        accessToken:(nullable NSString *)accessToken
-                      pendingToken:(nullable NSString *)pendingToken NS_DESIGNATED_INITIALIZER;
+                      pendingToken:(nullable NSString *)pendingToken
```

Comment:
I prefer switching the order of pendingToken and secret.

Reply:
Done.